code/modules/reagents/reagent_containers/food/drinks/bottle.dm

```diff
@@ -184,6 +184,14 @@
 		..()
 		reagents.add_reagent(/datum/reagent/ethanol/whiskey, 100)
 
+/obj/item/reagent_containers/food/drinks/bottle/ketchupwhiskey
+	name = "Crown Royal's Ketchup Flavored Whiskey"
+	desc = "A premium single-malt whiskey, gently matured then industralily ruined with artificial ketchup flavor."
```

Comment:
This sentence looks like something I wrote.

Reply:
is that good or bad?